mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/mail/JpaMessageMoveTest.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.james.mailbox.jpa.mail;
```

Comment:
Sorry, you forgot the compulsory Apache 2.0 license that should be on top of each file...